fish-rust/build.rs

```diff
@@ -0,0 +1,47 @@
+fn main() -> miette::Result<()> {
+    let rust_dir = std::env::var("CARGO_MANIFEST_DIR").expect("Env var CARGO_MANIFEST_DIR missing");
+    let target_dir =
+        std::env::var("FISH_RUST_TARGET_DIR").unwrap_or(format!("{}/{}", rust_dir, "target/"));
+    let fish_src_dir = format!("{}/{}", rust_dir, "../src/");
+
+    // Where cxx emits its header.
+    let cxx_include_dir = format!("{}/{}", target_dir, "cxxbridge/rust/");
```

Comment:
Thanks for the heads-up. I doubt fish will track upstream autocxx: its priorities are somewhat different (focused on long-term safe interop, instead of an aid to porting) so we'll continue to use our fork.